includes/configs/settings.php

```diff
@@ -401,6 +401,20 @@
 						'_parent'    => 'recaptcha_forms[]',
 						'_order'     => 30,
 					],
+
+					'recaptcha_version' => [
+						'label'   => esc_html__( 'Version', 'hivepress' ),
+						'type'    => 'select',
+						'default' => 'v2',
+
+						'options' => [
+							'v2' => esc_html__( 'reCAPTCHA V2', 'hivepress' ),
```

Comment:
Please don't include brand names to the translatable strings (even if this is required we use placeholders like %s), it's ok to use just numbers 2/3 here.

Reply:
Maybe it's also possible to check the reCaptcha version by the site key and secret they enter? e.g. if it differs with starter letters or another way, because users would 99% be confused which version they use. If reCaptcha now shows version 3 workflow by default for new users, then set version 3 as the default above.

Reply:
I don't think that's possible
When users copy their keys in the Google Console they can see which version of reCaptcha they are using